curl/test_package/conanfile.py

```diff
@@ -0,0 +1,21 @@
+from conans import ConanFile, CMake, tools
+
+
+class CurlTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if tools.cross_building(self.settings):
```

Comment:
This looks like a copy/paste mistake. No reason cross-compiling curl shouldn't work. Can you check some other test_package Conan recipes and compare?

Reply:
Most other packages do not have the "if tools.cross_building(self.settings):" check, perhaps it is best if we leave this part out?

Reply:
Yep, agreed